web/frontend/src/language/de.json

```
@@ -167,7 +167,7 @@
     "changeVote": "Sie können Ihre Stimme ändern, indem Sie einfach eine neue Stimme abgeben.",
     "pickCandidate": "Wählen Sie einen Kandidaten:",
     "voteSuccess": "Ihre Stimme wurde erfolgreich abgegeben!",
-    "voteSuccessful": "Abstimmung erfolgreich",
+    "voteSuccessful": "Stimmeabgabe erfolgreich",
```

```suggestion
    "voteSuccessful": "Stimmabgabe erfolgreich",
```
(definitely not trying to start a flame war on "proper" German though :see_no_evil:  )

---

You'll wind hands-down. Swiss-German is known to be it's own language...